docker/README.md

```diff
@@ -1,51 +1,66 @@
-# api3/airnode-artifacts
+# api3/airnode-packaging
 
-This is a Docker container for building all the packages in the monorepo and providing their build artifacts. Other
-package specific containers can use this one to obtain files necessary for their run without building or installing any
-additional dependencies.
+This is a Docker container for building all the packages in the monorepo, publishing the NPM packages (TODO) and
```

Comment:
```suggestion
This is a Docker container for building all the packages in the monorepo, publishing the NPM packages (TODO) and
```

These TODOs have a tendency to be forgotten, so I'd just avoid them and mention NPM publishing when implemented. Doesn't matter much to me though.